docs/.vitepress/theme/components/RunExamples.md

```diff
@@ -1,7 +1,22 @@
-To run the examples, execute the following script from your shell (from root folder), it will start a `cli` where you
+To run the examples, execute the following script from your shell (from root folder), it will start a CLI where you
 will select the framework and the pwa options:
 
 ```shell
 pnpm run examples
 ```
 
+
+<details>
+    <summary>Make sure you run <strong>pnpx build</strong> first</summary>
+
+```shell
+cd vite-plugin-pwa
+pnpm run install  # (if you need to)
+pnpm run build
+pnpm run examples
+```
+
+If you don't do `pnpx run build` first,
```

Comment:
should be: `pnpm run build`

Reply:
Doh! I meant to put `pnpm run build`, didn't meant to put `pnpx`, thanks for catching that!